### PR TITLE
AG-12606 Screenreader fixes

### PR DESCRIPTION
**Pull request by olegat**
https://ag-grid.atlassian.net/browse/AG-12606

### CODE REVIEW

File: packages/ag-charts-community/src/dom/elements.ts
Comment (olegat) [inline packages/ag-charts-community/src/dom/elements.ts:10]: Note: This `LabelIcon` allows 3 syntax modes:

```ts
{ label }         // text only
{ label, icon }   // text & icon
{ icon, altText } // icon only (with altText for screenreaders)
```

Specifying an `icon` without `altText`, or specifying both a `label` and `altText` will raise a compiler error.

File: packages/ag-charts-community/src/dom/elements.ts
Comment (olegat) [inline packages/ag-charts-community/src/dom/elements.ts]: This won't work for some property-attribute pairs like `ariaSetSize` / `aria-setsize` or `ariaPosInSet` / `aria-posinset`. We might want to have a more strict interface for the `Attrs` type.

Thoughts?

File: packages/ag-charts-locale/src/en-US.ts
Comment (lsjroberts) [inline packages/ag-charts-locale/src/en-US.ts:73]: Worth highlighting this with @JamesSwinton - https://github.com/ag-grid/ag-charts/pull/2478

File: packages/ag-charts-community/src/dom/elements.ts
Comment (lsjroberts) [reply packages/ag-charts-community/src/dom/elements.ts]: Agreed, the current `Attrs` type also allows function properties like `setEventListener` which is a bit leaky.

If we only care about targetting aria attributes, could we just replace the `aria` with `aria-` and not kebab the whole thing? Or are there other attrs we need to consider?

The reason I called it "missing" aria attrs is the typescript types for elements includes many attrs like `ariaLabel` but not `ariaControls`. Not sure if that is just our ts version being out of date or if it implies something else?

File: packages/ag-charts-locale/src/en-US.ts
Comment (olegat) [reply packages/ag-charts-locale/src/en-US.ts:73]: Noted. Thanks.

Comment (alantreadway) [reply packages/ag-charts-locale/src/en-US.ts:73]: We'll hold off merging #2478 until these new keys are merged.

File: packages/ag-charts-community/src/dom/elements.ts
Comment (olegat) [reply packages/ag-charts-community/src/dom/elements.ts]: AG-Grid uses wrappers to set aria attributes: 
https://github.com/ag-grid/ag-grid/blob/419770ca302ce87bc3c9a64b51a464c86132ab7c/community-modules/core/src/utils/aria.ts

We have something similar but it's not so widely used unfortunately:
https://github.com/ag-grid/ag-charts/blob/c2f6bb944c7dd8bce14803cdac6717a485e2537f/packages/ag-charts-community/src/util/attributeUtil.ts

It might be a good idea to start using `attributeUtil.ts` in more places sure as here.

**Review by alantreadway — APPROVED** [review]
LGTM

**Pull request merged**
